packages/blocks/src/page-block/default/utils.ts

```diff
@@ -0,0 +1,70 @@
+import type { BaseBlockModel } from '@blocksuite/store';
+import { assertExists, getBlockById } from '../../__internal__/utils';
+
+export const pick = (blocks: BaseBlockModel[], x: number, y: number) => {
```

Comment:
You need to add more description to explain this function.

As a reviewer, I want to know what it picks, what the `x` and `y` work, and what it returns, so you need to explain it.